pkg/osbuild/osbuild-exec.go

```diff
@@ -70,9 +70,13 @@ func RunOSBuild(manifest []byte, store, outputDirectory string, exports, checkpo
 
 	if result {
 		// try to decode the output even though the job could have failed
-		decodeErr := json.Unmarshal(stdoutBuffer.Bytes(), &res)
-		if decodeErr != nil {
-			return nil, fmt.Errorf("error decoding osbuild output: %v\nthe raw output:\n%s", decodeErr, stdoutBuffer.String())
+		if stdoutBuffer.Len() == 0 {
+			return nil, fmt.Errorf("osbuild did not return any output")
+		} else {
```

Comment:
The `else` here seems to be not needed because when the `if` is taken it will return and go prefers to avoid the "else" in this case (c.f. https://go.dev/doc/effective_go#if)